examples/selenoid-example/codecept.conf.js

```diff
@@ -0,0 +1,28 @@
+exports.config = {
+  tests: './*_test.js',
+  output: './output',
+  helpers: {
+    WebDriver: {
+      url: 'http://localhost',
+      browser: 'chrome',
+    },
+  },
+
+  plugins: {
+    selenoid: {
+      enabled: true,
+      name: 'selenoid',
+      deletePassed: true,
+      autoCreate: true,
+      autoStart: true,
+      sessionTimeout: '30m',
+      enableVideo: true,
+      enableLog: true,
```

Comment:
can we have default selenoid config? That would streamline the user's config instead of putting too much things.

Reply:
Not really sure about that - some will use Selenoid for browsers, other for videos. 
So all settings are necessary